src/native/libs/verify-entrypoints.sh

```diff
@@ -12,7 +12,8 @@ IFS=$'\n'
 dllList=()
 for line in $($nmCommand $1); do
   pattern='^[[:xdigit:]]+ T _?([[:alnum:]_]+)'
-  if [[ $line =~ $pattern ]]; then
+  # Individual platform-specific exports are excluded
+  if [[ $line =~ $pattern ]] && [[ "${BASH_REMATCH[1]}" != "SystemNative_GetUInt64OSThreadId" ]] && [[ "${BASH_REMATCH[1]}" != "SystemNative_TryGetUInt32OSThreadId" ]]; then
```

Comment:
Our strategy used in other places has been to implement a dummy entrypoint on all platforms, and just leave it unused on some.